packages/plugins/experimental/plugin-debug/src/components/Wireframe.tsx

```diff
@@ -13,7 +13,7 @@ export const Wireframe = ({ label, data, className }: { label?: string; data?: a
   const attendableAttrs = createAttendableAttributes(data);
   const { width, height, ref } = useResizeDetector();
   return (
-    <div ref={ref} className={mx('relative grow', className)} {...attendableAttrs}>
+    <div ref={ref} className={mx('relative grow min-h-[400px]', className)} {...attendableAttrs}>
```

Comment:
```suggestion
    <div ref={ref} className={mx('relative grow min-bs-96', className)} {...attendableAttrs}>
```

Unless it must be exactly `400px` in which case `min-bs-[400px]`.